recipes/platformio/meta.yaml

```diff
@@ -1,53 +0,0 @@
-{% set name = "platformio" %}
```

Comment:
Please don't remove other files in other recipes

Reply:
This file is still removed :frowning_face:

Reply:
@scopatz i added it back.